src/web/utils/urlHelpers.ts

```diff
@@ -1,10 +1,10 @@
 export const parseParticipantId = (participantId: string | undefined) => {
```

Comment:
@ssundahlTTD I changed my mind on this method - I'm now expecting an invalid `participantId` to be a genuine case when the user first signs in, see `ParticipantProvider.tsx`.